charts/ironic-standalone/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: ironic-standalone
+version: 1.1.27
+appVersion: wallaby
+description: Openstack Ironic chart for Kubernetes
+home: https://docs.openstack.org/ironic/latest/
+engine: gotpl
+maintainers:
+  - name: jakub-d
+    email: jakub.dlugolecki@gmail.com
+  - name: MaxRink
+    email: maximilian.rink@telekom.de
+  dependencies:
+  - name: mariadb
+    version: "9.x.x"
+    repository: "https://charts.bitnami.com/bitnami"
+    condition: mariadb.enabled
+  - name: rabbitmq
```

Comment:
Do you really need to use rabbitmq?

Reply:
Yes, if you want to scale your ironic instances for HA

Reply:
This is only correct for ironic-inspector, only if you need it in active/active HA, and only until we fix it upstream.